docker/Dockerfile

```diff
@@ -19,6 +23,10 @@ RUN yarn build output --okapi $OKAPI_URL --tenant $TENANT_ID
 
 # nginx stage
 FROM nginx:stable-alpine
+
+# Install latest patch versions of packages: https://pythonspeed.com/articles/security-updates-in-docker/
+RUN apk upgrade --no-cache
+
```

Comment:
Why is this `run apk` step totally separated from the other on line 12 when they seem releated? It feels stuck right in the middle of the nginx config. Can/should we move it up before line #17 to keep related things together?

Reply:
@zburke Because there are essentially two docker images that are getting created with this Dockerfile configuration.   The first one is an intermediate image used to build stripes and the second, final image is a runtime image.